### PR TITLE
HPCC-14616 Activity time sorting is alphabetical

### PR DESCRIPTION
**Pull request by GordonSmith**
Sort activity times correctly
Request row tracing (to console log)
Removed invalid activity/edge columns
Added started, running, finished styles
Fixed time conversion undefined issue
Simplify Visualization Debugging

fixes HPCC-14616

Signed-off-by: Gordon Smith gordonjsmith@gmail.com

### CODE REVIEW

**Comment by GordonSmith** [comment]
@miguelvazq Please Review

**Comment by richardkchapman** [comment]
@miguelvazq This seems to have stalled?

File: esp/src/eclwatch/GraphTreeWidget.js [hunk 1]
```diff
@@ -724,6 +724,16 @@ define([
             for (var i = 0; i < selectedGlobalIDs.length; ++i) {
                 this.global.displayProperties(selectedGlobalIDs[i], propertiesDom);
             }
+            var context = this;
+            if (selectedGlobalIDs.length) {
+                selectedGlobalIDs.filter(function (id) {
+                    return id && id.indexOf && id.indexOf("_") >= 0;
+                }).forEach(function (id) {
+                    WsWorkunits.WUCDebug(context.params.Wuid, "<debug:print edgeId='" + selectedGlobalIDs[0] + "'/>").then(function (response) {
+                        console.log(JSON.stringify(response));
```
Comment (miguelvazq) [inline esp/src/eclwatch/GraphTreeWidget.js]: This console.log really going to be useful for the user?

File: esp/src/eclwatch/WsWorkunits.js [hunk 2]
```diff
@@ -274,6 +274,18 @@ define([
...
         },
 
+        WUCDebug: function (wuid, command) {
+            return ESPRequest.send("WsWorkunits", "WUCDebug", {
+                request: {
+                    Wuid: wuid,
+                    Command: command
+                }
+            }).then(function (response) {
+                console.log(JSON.stringify(response));
```
Comment (miguelvazq) [inline esp/src/eclwatch/WsWorkunits.js]: console.log needed here?

File: esp/src/Visualization [hunk 3]
```diff
@@ -1 +1 @@
-Subproject commit a40abcd1b85efe2802f5c5cba1c3fa529ce01af3
+Subproject commit ae58f3281e6c48d30866cf7664db468b3f12cc14
```
Comment (miguelvazq) [inline esp/src/Visualization]: This commit part of the Visualization?

**Comment by miguelvazq** [comment]
Comments above.

**Comment by GordonSmith** [comment]
The console logs (as per the commit msg) are a stepping stone towards the row tracing feature.

**Comment by miguelvazq** [comment]
How about the visualization diff?

File: esp/src/Visualization (hunk 3 above)
Comment (GordonSmith) [inline esp/src/Visualization]: Yes - it means we are updating the version of the viz library.

**Comment by miguelvazq** [comment]
looks good then.

**Pull request merged**